packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -62,6 +72,12 @@ export interface PullRequestCommentBoxProps {
   isEditMode?: boolean
   onCancelClick?: () => void
   handleUpload?: HandleUploadType
+  principalProps: PrincipalPropsType
+  principalsMentionMap?: PrincipalsMentionMap
+  setPrincipalsMentionMap?: React.Dispatch<React.SetStateAction<PrincipalsMentionMap>>
+  preserveCommentOnSave?: boolean
+  buttonTitle?: string
```

Comment:
`buttonTitle` prop is to change the button text to "Create pull request" in PR compare page.